### ISSUE TITLE
Components of block of cheese not combining

### ISSUE THREAD

**Issue by MrDrummayoyo** [ValveSoftware/Dota2-Gameplay#14040]
### Item name

Block of Cheese

### Description

The three greater healing lotuses and cheese did not combine in my backpack, inventory, or any assortment. I did not drop and pick them up. None of the components were locked

![406509965_331186189725854_8838502221311881094_n](https://github.com/ValveSoftware/Dota2-Gameplay/assets/153041429/0df300be-7cf9-4f4c-9d19-d27f5f033d88)

### Example Match ID and Timestamp

7474561851 - 30:16

**Comment by NAlexP** [comment]
yeah, dota item combining is fuked like that, and has been for years. next time, don't keep items that you want combined in the backpack, and if issue persists, drop them and pick them back up
all of them
